app/src/components/ToDo.js

```diff
@@ -86,38 +112,28 @@ render (){
             </div>
              )
             } else {
-       return(
-            <div className={classes.root}>
-            <Grid container spacing={2}>
-              <Grid item xs={12} md={6}>
-                <Typography variant="h6" className={classes.title}>
-                  Tareas pendientes:
-                </Typography>
-                       
+       return(                       
                 <div className={classes.demo}>
-                  <List>
-                   {this.props.tareas.map((todo, index) =>  <ListItem key={index}> 
-                        <ListItemText primary={todo.description} />
+                
+                   <ListItem key={this.props.index}> 
+                        <ListItemText primary={this.props.tarea.description} />
                         <ListItemSecondaryAction>
-                        <IconButton onClick={() => this.handleEditToDo(todo)}>
+                        <IconButton onClick={() => this.handleEdit(this.props.tarea)}>
                            <EditIcon /> 
                           </IconButton>
-                          <IconButton edge="end" aria-label="delete" onClick={() => this.handleDeleteToDo(todo.id)}>
+                          <IconButton edge="end" aria-label="delete" onClick={() => this.handleDeleteToDo(this.props.tarea.id)}>
```

Comment:
same here

Reply:
Viendo mas en detalle, vos podrias acceder a this.props.tarea.id desde la funcion handleDeleteToDo.. no veo necesidad de pasarlo por parmetro